session/tidb.go

```diff
@@ -210,6 +210,9 @@ func finishStmt(ctx context.Context, se *session, meetsErr error, sql sqlexec.St
 	if err != nil {
 		return err
 	}
+
+	se.sessionVars.ClearTmpSystemVars()
```

Comment:
how about putting this to `ResetContextOfStmt()`?

Reply:
ok